### ISSUE TITLE
Error messages - red

### ISSUE THREAD

**Issue by jamie-benoit** [Nova-Scotia-Digital-Service/service-pattern-library#78]
In CNS design pattern, the red is 

![image](https://user-images.githubusercontent.com/13838127/144457199-56f7bbe3-fd16-46c8-b878-cf454dec9815.png)
Danger*
#cb1f17

and the error message, the red is 
![image](https://user-images.githubusercontent.com/13838127/144457535-e80f26ac-60f0-4227-b05e-a0ab37fa5638.png)
#A94442

Should error message be change to the Danger

**Comment by gabwhite** [comment]
Yes, there's also a CSS thing happening here where the colour is #cb1f17 darkened 5% in CSS, this isn't easy to replicate in XD (as far as I know)
Solution might be choosing a hex code for the red that doesn't need additional CSS tweaks
